pkg/test/fakes/gce_metadata_server/Makefile

```diff
@@ -0,0 +1,33 @@
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+.PHONY: build_and_push clean all
+
+MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+MD_PATH := $(dir $(MKFILE_PATH))
+IMG := gcr.io/istio-testing/fake-gce-metadata
+
+# NOTE: TAG should be updated whenever changes are made in this directory
+# This should also be updated in dependent components
+TAG := 1.0
+
+all: build_and_push clean
+
+build_and_push:
+	cd $(MD_PATH) && GOOS=linux GOARCH=amd64 go build main.go
```

Comment:
This should be a static build?

Reply:
I'm not using any cgo bits, as far as I'm aware (and it seems to work in my testing), but I guess there is no harm in being 100% sure.

Reply:
done.